README.md

```diff
@@ -54,6 +54,14 @@ function Routes() {
 
   A route will be imported synchronously if return true.
 
+- extensions: string[]
+
+  default: `['js', 'jsx', 'ts', 'tsx']`
+
+  Filename extensions which will be scanned and imported as route elements.
+  > You can use this plugin along with `@mdx-js/react` and its related deps
```

Comment:
Sorry, I think this sentance should not be added, it is not necessary.
Please remove it, thank you.

Reply:
Feel free to remove it at your will, it's your repository.
Since I didn't get it (which line), please modify as you wish. 🤔